openslides/motions/static/js/motions/diff.js

```diff
@@ -816,15 +816,22 @@ angular.module('OpenSlidesApp.motions.diff', ['OpenSlidesApp.motions.lineNumberi
             arr = splitArrayEntriesSplitSeparator(arr, ".");
             arr = splitArrayEntriesEmbedSeparator(arr, "\n", false);
 
-            return arr;
+            var arrWithoutEmptes = [];
+            for (var i = 0; i < arr.length; i++) {
+                if (arr[i] !== '') {
+                    arrWithoutEmptes.push(arr[i]);
+                }
+            }
+
+            return arrWithoutEmptes;
         };
 
-        this._outputcharcode = function (pre, str) {
+        this._outputcharcode = function (str) {
             var arr = [];
             for (var i = 0; i < str.length; i++) {
                 arr.push(str.charCodeAt(i));
             }
-            console.log(str, pre, arr);
+            console.log(arr.join(", "));
```

Comment:
This line was the whole point of this function, as this was just a helper function to debug different kind of whitespaces and other non-obvious-to-spot differences between two strings. But ok, I removed the whole function.